bookdown/04-technical/04-technical.Rmd

```diff
@@ -2,12 +2,39 @@
 
 This chapter gives an overview about technical details of the `r mlr_pkg("mlr3")` framework.
 
+**Parallelization**
+
 At first, some details about [Parallelization](#parallelization) and the usage of the `r cran_pkg("future")` are given.
-Parallelization refers to running multiple jobs simultaneously and is used to minimize necessary computer power. 
+Parallelization refers to the process of running multiple jobs simultaneously.
+Parallelization is used to minimize the necessary computer power.
+Irrespective, parallelization does not always minimize computer power.
+Algorithms consist of both sequential and paralizable parts. 
+Therefore, parallelization does not always alter performance in a positive substantial manner.
+In some cases, parallelization increases the duration of the entire undertaking.
+Summed up, this sub-chapter illustrates how and when to use parallelization in mlr3.
+
```

Comment:
Are there too many pointers in the paragraph why parallelization might be "bad"? It reads a bit as that parallelization might be more worse than good. Maybe we can use only one side sentence for this statement or put it in brackets?

Reply:
Yeah, I agree! It does sound overly negative. I changed it now.